src/azure-cli/azure/cli/command_modules/netappfiles/__init__.py

```diff
@@ -19,12 +19,23 @@ def __init__(self, cli_ctx=None):
 
     def load_command_table(self, args):
         from azure.cli.command_modules.netappfiles.commands import load_command_table
+        from azure.cli.core.aaz import load_aaz_command_table
+        try:
+            from . import aaz
+        except ImportError:
+            aaz = None
+        if aaz:
+            load_aaz_command_table(
+                loader=self,
+                aaz_pkg_name=aaz.__name__,
+                args=args
+            )
         load_command_table(self, args)
         return self.command_table
 
-    def load_arguments(self, command):
-        from azure.cli.command_modules.netappfiles._params import load_arguments
-        load_arguments(self, command)
+    # def load_arguments(self, command):
+        # from azure.cli.command_modules.netappfiles._params import load_arguments
+        # load_arguments(self, command)
```

Comment:
actually you should keep this function and an empty _params.py file for furture use.